low-level/files/opt/seagate/sspl/bin/sspl_config

```diff
@@ -89,7 +89,7 @@ config_sspl() {
     }
 
     # Put minion id, consul_host and consul_port in conf file
-    python3 $script_dir/salt_util.py $SSPL_CONF;
+    python3 $DIR_NAME/low-level/framework/utils/salt_util.py $SSPL_CONF;
```

Comment:
same comments, want deployment paths to remain same

Reply:
Done